Muvr/Models.bundle/slacking_model.layers.txt

```diff
@@ -1 +1,2 @@
-1200 500 100 25 2
\ No newline at end of file
+1200 500 100 25 2
+tanh tanh tanh tanh Sigm
```

Comment:
Did you update the model? The model I trained the other week was not trained with `tanh`.